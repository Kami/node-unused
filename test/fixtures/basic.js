// single unused variable

var a = 5;
