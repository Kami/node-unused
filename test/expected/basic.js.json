[
  {
    "name": "a",
    "loc": {
      "line": 3,
      "column": 4
    },
    "param": false
  }
]